packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/ContentTypeBuilderNav.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
I think everyone else is obeying this on the FE, we're using expect-error with a comment, which is basically what you're doing pretty much but that way we're not disabling rules and the code base is uniform 😊

Reply:
@Bassel17 how come you resolved this comment without a reply or apparent action?

Reply:
He mentioned @innerdvations and me this morning he took care of it, maybe it got lost with the conflicts we had to resolve, thanks for keeping an eye on it!